app/controllers/errors_controller.rb

```diff
@@ -38,11 +38,11 @@ def respond_to_error(status, message)
   end
 
   def serialize_errors(errors)
-    if request.headers['Accept'] == 'application/vnd.uktt.v2'
-      Api::V2::ErrorSerializationService.new.serialized_errors(errors)
-    else
-      Api::V1::ErrorSerializationService.new.serialized_errors(errors)
-    end
+    # if request.headers['Accept'] == 'application/vnd.uktt.v2'
```

Comment:
Commented this out for now because im pretty sure this can be removed but wanted to get some eyes on it first

Reply:
re-added as still needed

Reply:
This is one for @jebw, really, but I don't think we can remove it since we sometimes have v1 in the Accept header as per https://github.com/trade-tariff/trade-tariff-frontend/blob/main/lib/trade_tariff_frontend/request_forwarder.rb#L30

Reply:
Yeah think for now ive added back in so makes sense to leave and get the other change out 🙌